we are running from this directory: /lustre1/work/aurorasg/TNG
 the name of the job is: rotate
Th job ID is 3957941
The job was run on these nodes: compute-2-0-17
Number of nodes: 1
We are using 1 cores
We are using 1 cores per node
Total of 1 cores
Linux compute-2-0-17.foreman.hpc.ntnu.no 3.10.0-693.21.1.el7.x86_64 #1 SMP Wed Mar 7 19:03:37 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
Traceback (most recent call last):
  File "./src/check_rotation.py", line 104, in <module>
    subhalo_rotation(tng_run, test_name, 99, int(args.subhalo))
  File "./src/check_rotation.py", line 88, in subhalo_rotation
    group_cat = create_cat(new_cat_path)
  File "./src/check_rotation.py", line 23, in create_cat
    for filename in os.listdir(path):
FileNotFoundError: [Errno 2] No such file or directory: './data/tng-100-1/catalogues/test_runs/idun_3957941/'
Traceback (most recent call last):
  File "./src/check_rotation.py", line 104, in <module>
    subhalo_rotation(tng_run, test_name, 99, int(args.subhalo))
  File "./src/check_rotation.py", line 88, in subhalo_rotation
    group_cat = create_cat(new_cat_path)
  File "./src/check_rotation.py", line 23, in create_cat
    for filename in os.listdir(path):
FileNotFoundError: [Errno 2] No such file or directory: './data/tng-100-1/catalogues/test_runs/idun_3957941/'
Traceback (most recent call last):
  File "./src/check_rotation.py", line 104, in <module>
    subhalo_rotation(tng_run, test_name, 99, int(args.subhalo))
  File "./src/check_rotation.py", line 88, in subhalo_rotation
    group_cat = create_cat(new_cat_path)
  File "./src/check_rotation.py", line 23, in create_cat
    for filename in os.listdir(path):
FileNotFoundError: [Errno 2] No such file or directory: './data/tng-100-1/catalogues/test_runs/idun_3957941/'
Traceback (most recent call last):
  File "./src/check_rotation.py", line 104, in <module>
    subhalo_rotation(tng_run, test_name, 99, int(args.subhalo))
  File "./src/check_rotation.py", line 88, in subhalo_rotation
    group_cat = create_cat(new_cat_path)
  File "./src/check_rotation.py", line 23, in create_cat
    for filename in os.listdir(path):
FileNotFoundError: [Errno 2] No such file or directory: './data/tng-100-1/catalogues/test_runs/idun_3957941/'
Traceback (most recent call last):
  File "./src/check_rotation.py", line 104, in <module>
    subhalo_rotation(tng_run, test_name, 99, int(args.subhalo))
  File "./src/check_rotation.py", line 88, in subhalo_rotation
    group_cat = create_cat(new_cat_path)
  File "./src/check_rotation.py", line 23, in create_cat
    for filename in os.listdir(path):
FileNotFoundError: [Errno 2] No such file or directory: './data/tng-100-1/catalogues/test_runs/idun_3957941/'
Traceback (most recent call last):
  File "./src/check_rotation.py", line 104, in <module>
    subhalo_rotation(tng_run, test_name, 99, int(args.subhalo))
  File "./src/check_rotation.py", line 88, in subhalo_rotation
    group_cat = create_cat(new_cat_path)
  File "./src/check_rotation.py", line 23, in create_cat
    for filename in os.listdir(path):
FileNotFoundError: [Errno 2] No such file or directory: './data/tng-100-1/catalogues/test_runs/idun_3957941/'
Traceback (most recent call last):
  File "./src/check_rotation.py", line 104, in <module>
    subhalo_rotation(tng_run, test_name, 99, int(args.subhalo))
  File "./src/check_rotation.py", line 88, in subhalo_rotation
    group_cat = create_cat(new_cat_path)
  File "./src/check_rotation.py", line 23, in create_cat
    for filename in os.listdir(path):
FileNotFoundError: [Errno 2] No such file or directory: './data/tng-100-1/catalogues/test_runs/idun_3957941/'
Traceback (most recent call last):
  File "./src/check_rotation.py", line 104, in <module>
    subhalo_rotation(tng_run, test_name, 99, int(args.subhalo))
  File "./src/check_rotation.py", line 88, in subhalo_rotation
    group_cat = create_cat(new_cat_path)
  File "./src/check_rotation.py", line 23, in create_cat
    for filename in os.listdir(path):
FileNotFoundError: [Errno 2] No such file or directory: './data/tng-100-1/catalogues/test_runs/idun_3957941/'
Traceback (most recent call last):
  File "./src/check_rotation.py", line 104, in <module>
    subhalo_rotation(tng_run, test_name, 99, int(args.subhalo))
  File "./src/check_rotation.py", line 88, in subhalo_rotation
    group_cat = create_cat(new_cat_path)
  File "./src/check_rotation.py", line 23, in create_cat
    for filename in os.listdir(path):
FileNotFoundError: [Errno 2] No such file or directory: './data/tng-100-1/catalogues/test_runs/idun_3957941/'
